tests/OSPSuite.R.Tests/Services/ContainerTaskSpecs.cs

```diff
@@ -305,4 +305,37 @@ public void should_only_return_parameters_with_a_RHS_formula_defined()
          _result.ShouldOnlyContain(expected);
       }
    }
+
+   public class When_retrieving_the_is_formula_flag_for_a_given_path : concern_for_ContainerTask
+   {
+      [Observation]
+      public void should_return_the_expected_entries()
+      {
+         sut.IsExplicitFormulasFromPath(_simulation, pathFrom(_liver.Name, INTRACELLULAR, $"Vol{Constants.WILD_CARD}")).ShouldOnlyContain(false);
+
+         sut.IsExplicitFormulasFromPath(_simulation, pathFrom(_liver.Name, INTRACELLULAR, $"{Constants.WILD_CARD}Vol")).ShouldBeEmpty();
+
+         sut.IsExplicitFormulasFromPath(_simulation, pathFrom(Constants.WILD_CARD_RECURSIVE, Constants.WILD_CARD, Constants.Parameters.VOLUME)).ShouldOnlyContainInOrder(
+            true, false, false, false);
+      }
+   }
+
+
+   public class When_retrieving_the_base_unit_names_for_a_given_path : concern_for_ContainerTask
+   {
+      [Observation]
+      public void should_return_the_expected_entries()
+      {
+         sut.BaseUnitNamesFromPath(_simulation, pathFrom(_liver.Name, INTRACELLULAR, $"Vol{Constants.WILD_CARD}")).ShouldOnlyContain(_volumeLiverCell.Dimension.BaseUnit.Name);
```

Comment:
@PavelBal We have no fake stuff here.
in this case, you call you method and you verify that it does what you expect. Very simple